### DISCUSSION TITLE
Mode tailored for talks given by screenshare (e.g. Zoom)

### DISCUSSION THREAD

**Discussion by cpbl** [Cimbali/pympress#217] [Q&A] [answered]
For remote delivery of talks, one needs:

 - the audience view to be generated at normal/custom size, NOT scaled up to the entire screen, so that the conference software (e.g. Zoom) can be fed a reasonably-sized image for screen-sharing (ie my screen is unnecessarily large for rendering a PDF presentation)

 - for screenshare software which shares an entire window (rather than, say, a fixed portion of the desktop), the  audience view and presenter view should be  synchronized but in separate windows

Could pympress easily add these features?

# Environment
- Ubuntu 19.10 (can upgrade)

**Comment by Cimbali** [comment, marked as answer]
Hi @cpbl, I think you can do all of this out of the box.
- if you want pympress in windowed mode, simply press `f` or untick the `Presentation > Fullscreen` menu item. You can then resize the window as you like by dragging a corner like any other window.
- the audience view and presenter view are always synchronized and always in separate windows.

**Comment by cpbl** [comment]
This is awesome. Thank you. (I admittedly asked this question before upgrading my OS to Ubuntu 20.04, but have done so after your response). Kudos, all contributors. Both beamer-notes and next slide, non-full-screen windows, ... dreamy.
